src/bp/ui-admin/src/Pages/Account/ChangePassword.tsx

```diff
@@ -0,0 +1,128 @@
+import { Button, FormGroup, InputGroup, Intent } from '@blueprintjs/core'
+import React, { Component, Fragment } from 'react'
+import { RouteComponentProps } from 'react-router'
+import { Redirect } from 'react-router-dom'
+import { Alert, Card, CardBody, CardText, CardTitle } from 'reactstrap'
+
+import logo from '../../media/nobg_white.png'
+
+type Props = {
+  auth: any
+} & RouteComponentProps
+
+interface State {
+  loginUrl: string
+  email: string
+  password: string
+  newPassword: string
+  confirmPassword: string
+  error?: string
+}
+
+export default class ChangePassword extends Component<Props, State> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      ...props.location.state,
+      newPassword: '',
+      confirmPassword: '',
+      error: undefined
+    }
+  }
+
+  updatePassword = async () => {
+    const { email, password, newPassword, confirmPassword, loginUrl } = this.state
+
+    if (newPassword !== confirmPassword) {
+      return this.setState({ error: `Passwords don't match` })
+    }
+
+    this.setState({ error: undefined })
+
+    try {
+      await this.props.auth.login({ email, password, newPassword }, loginUrl)
```

Comment:
is the newPassword policy-validation completed in BE ?

Reply:
Should be, it was never touched since it was added a long time ago